src/download.jl

```diff
@@ -0,0 +1,41 @@
+"""
+    dataurl(dataid)
+
+This function takes in a dataset code of format "GADM_<Country Code>".\n
+Returns the URL of the gpkg dataset of the country
+"""
+function dataurl(dataid)
+    # Only accepts GADM Dataset Provider
+    if contains(dataid, "GADM")
+        provider, country_code = split(dataid, '_')
```

Comment:
You mean that the country code can contain `_` or `/` and that is why you replaced `/` by `_` in our convention? The slash character has this common usage out there to represent a "directory" structure that underscore doesn't have. I would go back to `/` if possible.

Reply:
DataDeps asks us for the dataset_name as the first parameter while declaring `DataDep()`

`dataset_name` is the unique identifier for a dataset. When we ask the user for the dataset code as `GADM/IND`, we will have to replace `/` with `_` while setting the dataset_name. This is because a folder cannot have `/` in the name.

This will result in the user having to register dataset using `GADM/IND` but asking for the dataset later by `GADM_IND`. I feel this might cause confusions.